### DISCUSSION TITLE
6x0 flats download issue

### DISCUSSION THREAD

**Discussion by LEFTYtheKID** [aderusha/DDD-Printable-Wall-Control-System#67] [General]
I am new to github so maybe its user error, but it appears the center, left, and right 6x0 flats needed for the wrench mount are unavailable for download. If I am missing something, I apologize but the normal download button I have used for every other part appears to not be available on those.

**Comment by leviable** [comment]
Yeah, common problem. For some reason GitHub has problems deciding what to do with the .stl files. 

Click the `raw`, then on the new page right click, `save as`, and erase the `.txt`.

**Reply by LEFTYtheKID** [reply]
Thanks!
